docs/src/components/LandingPage.module.css

```
@@ -0,0 +1,563 @@
+.page {
+  --landing-line: oklch(0.88 0 0);
+  --landing-line-strong: oklch(0.76 0 0);
+  --landing-surface: oklch(0.985 0 0);
+  --landing-surface-raised: oklch(1 0 0);
+  --landing-muted: oklch(0.43 0 0);
+  --landing-quiet: oklch(0.55 0 0);
+  position: relative;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)